src/libostree/ostree-repo-static-delta-processing.c

```diff
@@ -423,8 +423,9 @@ dispatch_bspatch (OstreeRepo *repo, StaticDeltaExecutionState *state, GCancellab
       struct bspatch_stream stream;
       stream.read = bspatch_read;
       stream.opaque = &opaque;
-      if (bspatch ((const guint8 *)g_mapped_file_get_contents (input_mfile),
-                   g_mapped_file_get_length (input_mfile), buf, state->content_size, &stream)
+      g_autofree guint8 *old = (guint8 *)g_mapped_file_get_contents (input_mfile);
```

Comment:
This is definitely wrong; the whole point of `GMappedFile` aka `mmap()` on Unix is that it's a view into memory owned by the kernel. It'd be useful to see the Coverity warning here, but regardless we need to skip it.

Really `g_mapped_file_get_contents` should have returned a `const char *`...but it'd be an API break now.